pkg/volume/volume_linux.go

```diff
@@ -46,7 +46,7 @@ func SetVolumeOwnership(mounter Mounter, fsGroup *int64, fsGroupChangePolicy *v1
 
 	fsGroupPolicyEnabled := utilfeature.DefaultFeatureGate.Enabled(features.ConfigurableFSGroupPolicy)
 
-	klog.Warningf("Setting volume ownership for %s and fsGroup set. If the volume has a lot of files then setting volume ownership could be slow, see https://github.com/kubernetes/kubernetes/issues/69699", mounter.GetPath())
+	klog.V(3).Infof("Setting volume ownership for %s and fsGroup set. If the volume has a lot of files then setting volume ownership could be slow, see https://github.com/kubernetes/kubernetes/issues/69699", mounter.GetPath())
 
 	// This code exists for legacy purposes, so as old behaviour is entirely preserved when feature gate is disabled
```

Comment:
nit: since you are making change in this file, can you please also fix this spelling issue:

```suggestion
	// This code exists for legacy purposes, so as old behavior is entirely preserved when feature gate is disabled
```

Reply:
I'm not sure that it worth to be fixed BTW: https://ell.stackexchange.com/questions/202548/what-is-the-difference-between-behavior-and-behaviour

Reply:
thanks for the info. Recommended based on VSCode spell checker =) I guess it uses American English than

Reply:
Ok, marking this as resolved.